tests/v3_api/test_auth.py

```diff
@@ -475,7 +479,13 @@ def enable_openldap_nestedgroup(username, token, expected_status=200):
         "userSearchAttribute": "uid|sn|givenName",
         "userSearchBase": USER_SEARCH_BASE,
         "serviceAccountPassword": SERVICE_ACCOUNT_PASSWORD
-      },
+    }
+
+    ca_cert = ldapConfig["certificate"]
+    ldapConfig["certificate"] = ca_cert.replace('\\n', '\n')
+
```

Comment:
Can we do this ca_cert.replace('\\n', '\n') one time, after you read the env variable. So that each method will not have to do this separately.

Reply:
The ca_certificate problem is not happened in reading env_var, since I printed the env_var, the env_var is correct. It happens after the env_var passed into json config fields, so the convertion is after that.